lib/jest.config.js

```diff
@@ -1,12 +1,12 @@
-module.exports = {
+export default {
   coverageDirectory: '.coverage',
   moduleDirectories: ['<rootDir>/src', 'node_modules'],
   moduleFileExtensions: ['js', 'ts', 'tsx'],
   moduleNameMapper: {
     // Map aliases defined in tsconfig.json to the absolute path of the module
     '^:/(.*)$': '<rootDir>/src/$1',
   },
-  resolver: '<rootDir>/jest/resolver.js',
+  resolver: '<rootDir>/jest/resolver.cjs',
```

Comment:
Why did the extension changed?

Reply:
Since we use `type: module` within `lib/package.json`. All `.js` files are considered as ES Modules but that is not the case the resolver script which must use common JS syntax. Using `.cjs` extension tells to jest to consider it as a common js script.